### DISCUSSION TITLE
Limit FPS in menu like in stable.

### DISCUSSION THREAD

**Discussion by Chrisx97** [ppy/osu#27636] [Ideas] [answered]
I've recently noticed that my CPU runs rather hot (compared to stable) while essentially just sitting idle in the menu on Lazer. 

Ryzen 5 5600 on a stock cooler hovers around 65-70c in Lazer menu/song select (looks like the frame-limit in menu is around 1000fps) 
on stable, menu temperatures are between 45-50c. (looks like the frame-limit here is about 288fps). 

Would really like to see an option to limit the frame-rate in the main menu to prevent this unnecessary load if possible.

**Comment by Shirou27** [comment]
Same to android. The temporary solution for this is to pause the music if you going to idle in the menu.

**Comment by peppy** [comment, marked as answer]
Hit ctrl+f7 to adjust the frame limiter temporarily.

**Reply by LePenner** [reply]
Are there any plans to implement this in the future for convenience’ sake? 

I stream and have a weak GPU (technically none) during gameplay it's fine, but it's quite the load in the menu and lags the stream. 
It's worst in multi menus.
